examples/todo-list/test/acceptance/todo-list-todo.acceptance.ts

```diff
@@ -29,82 +31,78 @@ describe('Application', () => {
     await todoListRepo.deleteAll();
   });
 
+  beforeEach(async () => {
+    persistedTodoList = await givenTodoListInstance();
+  });
+
   it('creates todo for a todoList', async () => {
-    const todoList = await givenTodoListInstance();
     const todo = givenTodo();
     const response = await client
-      .post(`/todo-lists/${todoList.id}/todos`)
+      .post(`/todo-lists/${persistedTodoList.id}/todos`)
       .send(todo)
       .expect(200);
 
     expect(response.body).to.containDeep(todo);
   });
 
-  it('finds todos for a todoList', async () => {
-    const todoList = await givenTodoListInstance();
-    const notMyTodo = await givenTodoInstance({
-      title: 'someone else does a thing',
+  context('when dealing with multiple persisted Todos', () => {
+    let notMyTodo: Todo;
+    let myTodos: Todo[];
+
+    beforeEach(async () => {
+      notMyTodo = await givenTodoInstance({
+        title: 'someone else does a thing',
+      });
+      myTodos = [
+        await givenTodoInstanceOfTodoList(persistedTodoList.id),
+        await givenTodoInstanceOfTodoList(persistedTodoList.id, {
```

Comment:
This code is creating the two instances serially. It's probably better to run these tasks in parallel:

```ts
myTodos = await Promise.all([
  givenTodoInstanceOfTodoList(persistedTodoList.id),
  givenTodoInstanceOfTodoList(persistedTodoList.id, {
    // ...
  }),
]);
```

Not a big deal though.